sdk/client_test.go

```diff
@@ -0,0 +1,67 @@
+package sdk
+
+import (
+	"net/http"
+	"testing"
+
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+const (
+	authHeader = "Authorization"
+	authPrefix = "Bearer "
+)
+
+func TestHeaders_Add(t *testing.T) {
+	t.Parallel()
+
+	Convey("given the sdk Headers struct contains a value for ETag", t, func() {
+		req := &http.Request{}
+		headers := &Headers{
+			ETag: "dsalfhjsadf",
+		}
+
+		Convey("when calling the Add method on Headers", func() {
+			headers.Add(req)
+
+			Convey("then an ETag header is set on the request", func() {
+				So(req.Header, ShouldBeEmpty)
```

Comment:
I think it would be good to add a TODO here e.g.
```suggestion
                                 //TODO Replace the 'ShouldBeEmpty' with the expected header when the ETag header has been implemented
				So(req.Header, ShouldBeEmpty)
```

Reply:
nope, the test will fail once eEag is implemented